packages/amplify-schema-validator/src/helpers/is-scalar-or-enum.ts

```diff
@@ -0,0 +1,53 @@
+import { EnumTypeDefinitionNode, Kind, TypeNode } from 'graphql';
+
+type ScalarMap = {
+  [k: string]: 'String' | 'Int' | 'Float' | 'Boolean' | 'ID';
+};
+export const STANDARD_SCALARS: ScalarMap = {
+  String: 'String',
+  Int: 'Int',
+  Float: 'Float',
+  Boolean: 'Boolean',
+  ID: 'ID',
+};
+
+const OTHER_SCALARS: ScalarMap = {
```

Comment:
Does Amplify support these?

Reply:
The code is lifted from https://github.com/aws-amplify/amplify-category-api/blob/main/packages/graphql-transformer-common/src/definition.ts#L35-L38, so I think it's safer to keep this in order to maintain parity.